index.html

```diff
@@ -2406,13 +2406,23 @@ <h3>HTML Element Role Mappings</h3>
             <tr tabindex="-1" id="el-output">
               <th><a data-cite="HTML">`output`</a></th>
               <td class="aria"><a class="core-mapping" href="#role-map-status">`status`</a> role</td>
-              <td class="ia2"><div class="general">Use WAI-ARIA mapping</div></td>
-              <td class="uia">
+              <td class="ia2">
                 <div class="general">Use WAI-ARIA mapping</div>
-                <div class="ctrltype"> <span class="type">Localized Control Type:</span> `"output"`</div>
+                <div class="relations">
+                  <span class="type">Relations:</span> `IA2_RELATION_LABELLED_BY` with associated `label` element
+                </div>
+              </td>
+              <td class="uia"><div class="general">Use WAI-ARIA mapping</div></td>
+              <td class="atk">
+                <div class="general">Use WAI-ARIA mapping</div>
+                <div class="relations">
+                  <span class="type">Relations:</span> `ATK_RELATION_LABELLED_BY` with associated `label` element
+                </div>
+              </td>
+              <td class="ax">
+                <div class="general">Use WAI-ARIA mapping</div>
+                <td class="ax">`AXDescription: value from associated `label` element subtree.</td>
```

Comment:
For future reference, pretty much all instances to `AXTitle` or `AXDescription` should be replaced with `AXLabel`. There was a legacy distinction in the original Mac OS X Accessibility Protocol that proved irrelevant. 
```suggestion
                <td class="ax">`AXLabel: value from associated `label` element subtree.</td>
```

Reply:
It also may be worth considering whether to list these using a more common developer reference, like `accessibilityLabel`. If @aleventhal or other non-Apple implementors have a preference, please advise. Thanks.
https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel

Reply:
Unless you object, @cookiecrook, I'm going to leave this as is since there's a larger find and replace issue I filed to fix this more broadly (#450).

That way, if this is instead changed to `accessibilityLabel`, then this instance doesn't need to be changed again.

Reply:
sort of related, you should fix the missing backtick for AXDescription